contracts/sei-tester/src/contract.rs

```diff
@@ -183,25 +183,27 @@ pub fn change_admin(
 // set coin metadata for a tokenfactory denom.
 pub fn set_metadata(
     _deps: DepsMut<SeiQueryWrapper>,
-    _env: Env,
+    env: Env,
     _info: MessageInfo,
 ) -> Result<Response<SeiMsg>, StdError> {
     let test_metadata = Metadata {
         description: "Token Metadata".to_string(),
-        base: "token".to_string(),
-        display: "TOKEN".to_string(),
-        name: "token_name".to_string(),
-        symbol: "TOKEN".to_string(),
+        base: "factory/".to_string() + env.contract.address.to_string().as_ref() + "/subdenom",
```

Comment:
nit, can we assign this to a variable and then reference that in the metadata definiton? (see above function for `create_denom` for example

Reply:
Updated on latest commit